domains/cosintan.json

```
@@ -1,9 +1,15 @@
 {
-  "owner": {
-    "username": "cosintan-tg",
-    "email": "cosintan.tg@gmail.com"
-  },
-  "records": {
-    "CNAME": "cname.vercel-dns.com"
-  }
+    "owner": {
+        "username": "cosintan-tg",
+        "email": "cosintan.tg@gmail.com"
+    },
+    "records": {
+        "CNAME": "cname.vercel-dns.com"
+    },
+    "services": {
+        "vercel": "vc-domain-verify=cosintan.is-a.dev,b33d7124eaba6f7dca28",
+        "discord": "dh=cc6050f09d0f1563f4dd1b168879b9b03ad5e57f",
+        "MX": ["mx01.mail.icloud.com", "mx02.mail.icloud.com"],
+        "TXT": ["apple-domain=LiWFOklmiBC4xnoS", "v=spf1 include:icloud.com ~all"]
+    }
```

```suggestion
    "records": {
        "CNAME": "cname.vercel-dns.com",
        "MX": ["mx01.mail.icloud.com", "mx02.mail.icloud.com"],
        "TXT": ["apple-domain=LiWFOklmiBC4xnoS", "v=spf1 include:icloud.com ~all"]
    },
    "proxied": true
```

"services" is disabled/discontinued , read this guide for connecting your Domain to Discord -> https://docs.is-a.dev/guides/discord-verification/
invalid Domain Structure please read -> https://docs.is-a.dev/domain-structure/